integrations-and-supported-tools/pytorch/scripts/Neptune_Pytorch_Support_Advanced.py

```diff
@@ -0,0 +1,182 @@
+from numpy.random import permutation
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import torchvision
+import matplotlib.pyplot as plt
+from torchvision import datasets, models, transforms
+import neptune.new as neptune
+import numpy as np
+
+def main():
+    # Step 1: Initialize Neptune and create new run
+    run = neptune.init(project='common/pytorch-integration', api_token='ANONYMOUS', source_files=['*.py'])
+
+    # Helper functions
+    def save_model(model, name ='model.txt'):
```

Comment:
I think in the examples it is better to keep the context close -> have model saving (both the helper and neptune code) next to each other for ease of reading

Reply:
I will be weird. I think helper functions are at the top of the script or in a external file(which I prefer)

Reply:
I created a script called helpers.py and put all the functions there. It makes it easier to read and navigate. 

If the user want to see a particular helper they can go and check there.